R/utils.R

```diff
@@ -23,3 +23,24 @@ check_proper_color <- function(color) {
     invisible(color)
   }
 }
+
+#' Generates random id of element
```

Comment:
Can we provide more details when to use it and example?

Reply:
This is hidden function, not to see by the user, actually examples of hidden function cause problem when doing checks for CRAN, but I elaborated in the description.